recordriver/driver.go

```diff
@@ -156,6 +156,18 @@ func (*Response) Close() error {
 	return nil
 }
 
+func (r *Response) clone() *Response {
+	var c Response
+	c.Cols = make([]string, len(r.Cols))
+	copy(c.Cols, r.Cols)
```

Comment:
This is essentially `r.Cols[:]`.

Reply:
Indeed, it didn't come up when I googled

I'm surprised that this worked with `Data[:]`

Reply:
Thanks!